db/seeds.rb

```diff
@@ -5,20 +5,21 @@
 user1 = User.create!(
   name: 'user1',
   email: 'user1@example.com',
-  password: 'hello123',
+  password: 'hello123', 
   password_confirmation: 'hello123'
 )
 
-user1.skip_confirmation!
-user1.save
-
 user2 = User.create!(
-  name: "user2",
-  email: "user2@example.com",
-  password: "hello123",
+  name: 'user2',
+  email: 'user2@example.com',
+  password: 'hello123', 
   password_confirmation: 'hello123'
 )
 
+
+user1.skip_confirmation!
+user1.save
+
 user2.skip_confirmation!
 user2.save
 
```

Comment:
When de-duplicating very similar code like the `User` creation code in seeds.rb, it can be helpful to introduce variables to make the code as similar as possible.

Here's what the duplicated code looks like in `seeds.rb` currently:

```ruby
user1 = User.create!(
  name: 'user1',
  email: 'user1@example.com',
  password: 'hello123',
  password_confirmation: 'hello123'
)

user2 = User.create!(
  name: 'user2',
  email: 'user2@example.com',
  password: 'hello123',
  password_confirmation: 'hello123'
)


user1.skip_confirmation!
user1.save

user2.skip_confirmation!
user2.save
```

As a first step, I'm going to keep the duplication, but introduce a local variable `i` to represent the variable part of the process:

```ruby
# Start with i = 1:
i = 1
user1 = User.create!(
  name: "user#{i}",
  email: "user#{i}@example.com",
  password: 'hello123',
  password_confirmation: 'hello123'
)

# Increment i to 2:
i = 2
user2 = User.create!(
  name: "user#{i}",
  email: "user#{i}@example.com",
  password: 'hello123',
  password_confirmation: 'hello123'
)


user1.skip_confirmation!
user1.save

user2.skip_confirmation!
user2.save
```

Review the above and note how similar the 2 calls to `User.create!(...)` are now.

In the next step, I'm going to reuse a variable named `user`, to replace the 2 variable names `user1` and `user2`. This will involve moving the `skip_confirmation!` and `save` calls for the first user up a few lines:


```ruby
i = 1
user = User.create!(
  name: "user#{i}",
  email: "user#{i}@example.com",
  password: 'hello123',
  password_confirmation: 'hello123'
)
user.skip_confirmation!
user.save

i = 2
user = User.create!(
  name: "user#{i}",
  email: "user#{i}@example.com",
  password: 'hello123',
  password_confirmation: 'hello123'
)
user.skip_confirmation!
user.save
```

Review the above and note how even more similar the 2 sections of code are. The only lines that are not duplicate lines are the `i = 1` and `i = 2` lines.

Now we've got 2 almost identical code snippets for User creation. The next step is to remove the duplication entirely. Try to come up with a way to do this using `.each`.